StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -191,6 +191,9 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
 #pragma endregion CreateScene1
 	char sceneName[10] = "scene0";
 	sceneManager->SetSceneToDraw(sceneManager->GetScene(sceneName));
+
+    Input::GetInstance().RegisterDevices(window->GetHwnd());
```

Comment:
Do this higher up since this is an init() type function. Would suggest right under engine.init().